apps/worker/src/queue.ts

```
@@ -0,0 +1,367 @@
+  env: Env
+): Promise<void> {
+  try {
+    // Get markdown content from R2
+    const r2Key = `users/${userId}/notes/${noteId}/content.md`;
+    const markdownObject = await env.R2.get(r2Key);
+
+    if (!markdownObject) {
+      throw new Error(`Markdown not found for note ${noteId}`);
+    }
+
+    const markdown = await markdownObject.text();
```

ESLint

[error] 'markdown' is assigned a value but never used.

(no-unused-vars)